packages/admin-ui/src/page/page-header-button.ts

```diff
@@ -0,0 +1,37 @@
+import { createComponent, useElement } from '@vtex/admin-ui-react'
+
+import type { ButtonOptions } from '../button'
+import { Button } from '../button'
+import * as style from './page.style'
+
+/**
+ * Page header button component
+ *
+ * @example
+ * import { PageHeader, PageHeaderTop, PageHeaderActions, PageHeaderButton } from "@vtex/admin-ui"
+ *
+ * <PageHeader>
+ *  <PageHeaderTop>
+ *    <PageHeaderTitle>
+ *      Product #123
+ *    </PageHeaderTitle>
+ *    <PageHeaderActions>
+ *      <PageHeaderButton>Create</PageHeaderButton>
+ *      <PageHeaderButton variant="critical">Delete</PageHeaderButton>
+ *    </PageHeaderActions>
+ *  </PageHeaderTop>
+ * </PageHeader>
+ */
+export const PageHeaderButton = createComponent<typeof Button, ButtonOptions>(
```

Comment:
Avoid using `createComponent` with a component created using `createComponent` :inception:. This can lead to some types and behavior errors since we don't cover this scenario
Do something like this instead
```jsx
export const PageHeaderButton = forwardRef((props: ButtonProps, ref: Ref<HTMLButtonElement>) => {
  const { size="large", ...buttonProps } = props
  return <Button ref={ref} size={size} {...buttonProps} />
})
```

Reply:
Nice! Thanks for the tip @lucasaarcoverde

Reply:
Done!